### PR TITLE
feat(Select): add disabled flag for component

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Adds isDisabled flag to select.

**Refer to issue**: #2366

### CODE REVIEW

**Comment by kmcfaul** [comment]
Requires core to be updated to v2.26 to get styling

File: packages/patternfly-4/react-core/src/components/Select/Select.tsx [hunk 1]
```diff
@@ -28,6 +28,8 @@ export interface SelectProps
   isGrouped?: boolean;
   /** Display the toggle with no border or background */
   isPlain?: boolean;
+  /** Flad to inficate if select is disabled */
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/Select/Select.tsx]: typo

File: packages/patternfly-4/react-core/src/components/Select/Select.md [hunk 2]
```diff
@@ -325,6 +407,84 @@ class TypeaheadSelectInput extends React.Component {
...
+
+```js
+import React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+class TypeaheadSelectInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.options = [
+      { value: 'Alabama', disabled: false },
```
Comment (jschuler) [inline packages/patternfly-4/react-core/src/components/Select/Select.md]: I'd remove the disabled keys from the options array in the two added examples for clarity purposes

Comment (tlabaj) [reply packages/patternfly-4/react-core/src/components/Select/Select.md]: I was thinking they same thing.

**Comment by tlabaj** [comment]
@jschuler ,  I think the examples are fine as is.  They match core.

**Pull request merged**